### PR TITLE
Added 3 new maps, including two 3v3 maps

### LINKED ISSUE #397: Create new Battleground Arenas for 1v1s and two 3v3 arenas.
**Describe the solution you'd like**
Required art:
- 3 battleground floors
- 3 new wall art examples

Required code:
- At least kind of cleanly handle this in Battleground.cs. It is going to be somewhat messy, but at least make it kind of reasonable.
- Make the rendering engine support the fact that the size of the battleground can now change (it needs to be bigger when there are 3v3s.
- Create new wall layouts for the new maps.
- Create spawn positions for new maps, and implement system to list spawn positions for each map in a reasonably clean way.
- Create a MapType class or data type.
- Create a selector in training arena.
- Pick a random map of the appropriate size in the battle arena.

### PR DESCRIPTION
**Pull request by SecondThread**
**Description**
Added maps: Hex (1v1), Canden (3v3), and Lunar (3v3). Each have different art and wall layouts. There is a also a second type of wall used in the bigger maps. Also updated the training arena to have a dropdown to select the map.

Here are the final map layouts:
Classic:
image

Hex:
image

Canden:
image

Lunar:
image

**Resolves These Issues**
Resolves #397 

**Type of change**
Check options that are relevant.

- [ ] Bug fix (fixes a bug issue)
- [x] New feature (adds functionality)
- [ ] This fix or feature will cause existing functionality to not work as expected. (Please elaborate in Description.)

**Steps to Verify Functionality**
Make a bulleted/numbered list of steps to verify this feature/bugfix. Include screenshots if necessary.

**Final Checklist:**
Go down the list and check them off.

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my own code
- [x] My changes pass Flow, build without errors and (if applicable) pass Jest tests.
- [ ] This change requires a update in the design document (if applicable)

### CODE REVIEW

**Comment by Derplime** [comment]
Is the Classic map going to be referenced as DIRT when this is used? Or would you like to just edit the map enum in your branch?

**Comment by SecondThread** [comment]
internally it is called "DIRT". I just say classic because Dirt isn't a super attractive name, although the map is pretty good

**Comment by Derplime** [comment]
> internally it is called "DIRT". I just say classic because Dirt isn't a super attractive name, although the map is pretty good

Ah alright thanks

File: src/battlearena/BattleArena.js [hunk 1]
```diff
@@ -68,6 +69,8 @@ class BattleArena extends React.Component<Props, State> {
 		prepareMatchAPICall(myTank, player, matchId => {
 			console.log('Successfully prepared match with id: '+matchId);
 			setMatchForBattleground(matchId);
+			//TODO: select an appropriate arena depending on the match
+			setBattlegroundArena('DIRT');
```
Comment (Derplime) [inline src/battlearena/BattleArena.js:73]: Mostly a reminder to update this

**Review by Derplime — APPROVED** [review]
Looks good. Doubled down on that TODO when it has to be updated but other than that you're good.

**Pull request merged**